config/v1/0000_10_config-operator_01_infrastructure.crd.yaml

```diff
@@ -142,6 +143,7 @@ spec:
                   enum:
                     - HighlyAvailable
                     - SingleReplica
+                    - External
```

Comment:
are we actually consistent about infra?  

in hypershift we have some infra components that run external (olm) and some that run on cluster (registry) right?

Reply:
That is correct. From an operator point of view, I don't think the External value makes sense for infra topology (Unless we really externalize all of infra). It is meant to apply only to control plane topology for now. However, not sure that we can express that through the API as it is.

Reply:
So "external" will not be a valid value for status.infrastructureTopology ?

I guess can understand not creating another Type to split them, so i guess we can live with this but it is somewhat unfortunate that it confusingly implies it is valid, even if practice we'd never actually set it to that value.